src/main/thrift/content/v1.thrift

```diff
@@ -0,0 +1,763 @@
+namespace scala com.gu.contentapi.client.model.v1
+
+struct CapiDateTime {
```

Comment:
As I mentioned the other day, it might be nice to provide some helpers to make this clunky wrapper a bit easier for users to work with.

e.g. (untested, may not compile):

```scala
package com.gu.contentapi.client.model.v1

import org.joda.time.DateTime

object CapiModelEnrichment {

  implicit class RichCapiDateTime(val cdt: CapiDateTime) extends AnyVal {
    def toJodaDateTime: DateTime = new DateTime(cdt.dateTime)
  }

  implicit class RichJodaDateTime(val dt: DateTime) extends AnyVal {
    def toCapiDateTime: CapiDateTime = CapiDateTime.apply(dt.getMillis)
  }

}
```